src/processes/news-process.js

```diff
@@ -1,213 +1,250 @@
+    });
+
+    let domain_id;
+    let name;
+    let info;
+    let domain_image;
+    let created_domain;
+    if (getDomain) {
+      domain_id = getDomain.dataValues.domain_page_id;
+      name = getDomain.dataValues.domain_name;
+      info = getDomain.dataValues.description;
+      domain_image = getDomain.dataValues.logo;
+      created_domain = getDomain.dataValues.created_at;
+    } else {
+      const crawls = await axios.get(`${resp.request.protocol}//${resp.request.host}`, {
+        headers: {'User-Agent': 'bettersocial'}
+      });
+      const $ = cheerio.load(crawls.data);
+      const logo = $('meta[property="og:image"]').attr('content') || '';
+      const description = $('meta[property="og:description"]').attr('content') || '';
+      const domainId = uuidv4();
+      const data = {
+        domain_page_id: domainId,
+        domain_name: removeWww,
+        short_description: description.length > 254 ? description.substring(0, 230) : description,
+        logo,
+        ...dateCreted
+      };
+
+      await DomainPage.create(data);
+
+      domain_id = data.domain_page_id;
+      name = data.domain_name;
+      info = data.description;
+      domain_image = data.logo;
+      created_domain = data.created_at;
     }
-}
 
-const putMainFeed = async (job, name, logo, created, data, newsLinkFeedId) => {
-    const { putStream } = require('../services');
+    const queueOptions = {
+      limiter: {
+        max: 300,
+        duration: 60 * 1000 // 60k ms = 1 minute
+      }
+    };
+
+    credderScoreQueue.add(
+      {
+        domainName: removeWww
+      },
+      queueOptions
+    );
+
+    return {domain_id, name, info, domain_image, created_domain};
+  } catch (error) {
+    console.error(error);
+    return error;
+  }
+};
 
-    try {
-        const set = {
-            post_type: 2,
-            og: {
-                domain: name,
-                date: created,
-                domainImage: logo,
-                title: data.title,
-                description: data.description,
-                image: data.image,
-                url: data.news_url,
-                domain_page_id: data.domain_page_id,
-                news_link_id: data.news_link_id,
-                news_feed_id: newsLinkFeedId
-            }
-        }
-        await putStream(job.id_feed, set);
-        console.info(`updated main_feed:${job.id_feed}`)
-    } catch (error) {
-        console.info(error)
-    }
-}
+const putMainFeed = async (job, name, logo, created, data, newsLinkFeedId) => {
+  try {
+    const set = {
+      post_type: 2,
+      og: {
+        domain: name,
+        date: created,
+        domainImage: logo,
+        title: data.title,
+        description: data.description,
+        image: data.image,
+        url: data.news_url,
+        domain_page_id: data.domain_page_id,
+        news_link_id: data.news_link_id,
+        news_feed_id: newsLinkFeedId,
+        content_published_at: data?.content_published_at
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The optional chaining operator `?.` is used here, which is unnecessary because the `data` object should always be defined at this point in the code. If it's not, there are bigger issues that need to be addressed.
```diff
-        content_published_at: data?.content_published_at
+        content_published_at: data.content_published_at
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->